src/test/java/core/basesyntax/service/FileReaderServiceTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import java.io.File;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static final String testTransactionFilePath = "src/main/resources/transaction.csv";
```

Comment:
how we should write constants?

Reply:
Maven told to do this

Reply:
Ye, right. But Maven think what "TEST_TRANSACTION_FILE_PATH" is wrong